hail.md

```diff
@@ -385,6 +385,82 @@ git merge upstream/main  # Potentially resolve any conflicts.
 git push origin upstream  # Create a PR as usual.
 ```
 
+### Summary of steps when merging upstream changes
+
+1. Check if your .gitconfig includes:
+
+   ```bash
+   [merge]
+      conflictstyle = diff3
+   [diff]
+      colorMoved = true
+      indentHeuristic = true
+      tool = difftastic
+      wordRegex = \\w+|\\W
+      #??? wsErrorHighlight = new
```

Comment:
Only really important line here is `merge.conflictstyle = diff3` (or maybe even suggest `zdiff3`).

`diff.colorMoved` is generally useful but not really too important for doing these merges. And the rest are specific to my environment or the default anyway.

Reply:
Thanks John! I had updated the file.